src/routes/transactions/transactions-history.imitation-transactions.controller.spec.ts

```diff
@@ -100,1393 +105,2586 @@ describe('Transactions History Controller (Unit) - Imitation Transactions', () =
     await app.close();
   });
 
-  function getImitationAddress(address: `0x${string}`): `0x${string}` {
-    // + 2 is to account for the '0x' prefix
-    const prefix = address.slice(0, prefixLength + 2);
-    const suffix = address.slice(-suffixLength);
-    const imitator = `${prefix}${faker.finance.ethereumAddress().slice(prefixLength + 2, -suffixLength)}${suffix}`;
-    return getAddress(imitator);
-  }
-  const chain = chainBuilder().build();
-  const safe = safeBuilder().build();
+  describe('Event spoofing', () => {
```

Comment:
I'd suggest hiding whitespace to review this. The test content has not drastically changed for the existing tests, but newer value-tolerance-focused tests have been added.